python/Cargo.toml

```diff
@@ -31,7 +31,7 @@ libc = "0.2"
 tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync"] }
 rand = "0.7"
 pyo3 = { version = "0.13.2", features = ["extension-module"] }
-datafusion = { git = "https://github.com/apache/arrow-datafusion.git", rev = "c3fc0c75af5ff2ebb99dba197d9d2ccd83eb5952" }
```

Comment:
This forces Python-datafusion to always use the latest datafusion release, which is not necessarily what we always want.

We agreed to keep using frozen hashes and perform manual changes. The rational is that releases of datafusion-python should be pinned to a specific version of DataFusion, so that we can release independently.

Reply:
Ah ok. Let's do it via a hash change then for now?